spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -970,6 +1015,7 @@ public void populateSparkWebUrl(InterpreterContext ctx) {
         infos.put("url", sparkUrl);
         logger.info("Sending metainfos to Zeppelin server: {}", infos.toString());
```

Comment:
Should we put the logging in the if block ? And what about else block ? Will that happen ?